src/ServerListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Panthère project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Panthere;
+
+use PHPUnit\Framework\TestListener;
+use PHPUnit\Framework\TestListenerDefaultImplementation;
+use PHPUnit\Framework\TestSuite;
+
+class ServerListener implements TestListener
+{
+    use TestListenerDefaultImplementation;
+
+    public function startTestSuite(TestSuite $suite): void
+    {
+        echo "Starting Panthere server for test suite {$suite->getName()}...\n";
+        PanthereTestCase::stopServerOnTeardown();
+        PanthereTestCase::startWebServer(
+            getenv('PANTHERE_LISTENER_SERVER_DIR') ?: null,
```

Comment:
We use `$_SERVER` everywhere.

Reply:
And passing arguments should be avoided because env vars are already handled in the called method.